kotlin-xml-builder/build.gradle.kts

```diff
@@ -1,44 +1,21 @@
 plugins {
-	kotlin("jvm")
-	`maven-publish`
-	signing
 	id("org.jlleitschuh.gradle.ktlint")
-}
-
-val kotlinVersion: String by rootProject.extra
-
-tasks {
-	val jar by getting(Jar::class)
-
-	register<Jar>("sourceJar") {
-		from(sourceSets["main"].allSource)
-		destinationDirectory.set(jar.destinationDirectory)
-		archiveClassifier.set("sources")
-	}
+	conventions.publishing
+	conventions.`kotlin-jvm`
 }
 
 dependencies {
-	compileOnly(kotlin("stdlib", kotlinVersion))
-	compileOnly(kotlin("reflect", kotlinVersion))
+	compileOnly(kotlin("reflect"))
 	implementation("org.apache.commons:commons-lang3:3.5")
 
 	testImplementation("junit:junit:4.13.1")
-	testImplementation(kotlin("reflect", kotlinVersion))
```

Comment:
This dependency is needed for one test to run correctly. Without it `OrderedNodesTest.correctOrder` fails because reflection libs are not available.

Reply:
Ah yes, that's right. I'll add it back.